static/canvas.js

```diff
@@ -297,20 +333,50 @@ class Canvas {
                 this.boxes[i].draw(ctx);
             }
 
-            /*Draws border for selected box. */
+            /* Draws border for selected box. */
+
             if (this.selection != null) {
-                ctx.strokeStyle = this.shadeColor(this.selection.thing.fill, -20);
-                ctx.lineWidth = this.selectionWidth;
-                var mySel = this.selection;
-                ctx.strokeRect(mySel.x, mySel.y, mySel.w, mySel.h);
-
-                /* Handles corners. */
-                var fillSize = this.selectionCorner;
-                ctx.fillStyle = "#000000";
-                ctx.fillRect(mySel.x - 2, mySel.y - 2, fillSize, fillSize);
-                ctx.fillRect(mySel.x - 3 + mySel.w, mySel.y - 2, fillSize, fillSize);
-                ctx.fillRect(mySel.x - 2, mySel.y - 2 + mySel.h, fillSize, fillSize);
-                ctx.fillRect(mySel.x - 3 + mySel.w, mySel.y - 2 + mySel.h, fillSize, fillSize);
+                for (var box of this.boxes) {
```

Comment:
In the old way, if you created a key frame in frame 500 and had it selected, if you moved to frame 200, the outline would still be there.